### DISCUSSION TITLE
#Vscode  Configure Runtime for Projects page show：No project detected yet. Please refresh later if Java extension is importing your projects.

### DISCUSSION THREAD

**Discussion by 9zyx3** [redhat-developer/vscode-java#3425] [Q&A]
I'am just a beginner
My vscode was a mess, so I reinstall it.
But I have a new problem : as the title suggests.
<img width="770" alt="屏幕截图 2023-12-10 153239" src="https://github.com/redhat-developer/vscode-java/assets/144330794/0b33fdd4-8567-4fec-89d7-21e53f59c484">
this is my setting.json:
<img width="770" alt="屏幕截图 2023-12-10 152415" src="https://github.com/redhat-developer/vscode-java/assets/144330794/940faf97-3e6e-4836-9d35-544124d6afb1">
and my java version:
<img width="867" alt="屏幕截图 2023-12-10 152407" src="https://github.com/redhat-developer/vscode-java/assets/144330794/83d0472a-7c10-44f1-84ab-fee4b2fe2da8">
Please tell me why and give me suggestions , thank you!

**Comment by BillyChen123** [comment]
I have the same question with you. Have you solved this problem now?

**Comment by marand85** [comment]
Hi everyone! I had the same problem and just fixed it. Finally! After uninstalling, reinstalling everything, all the add-ons, finally solved the problem and the java project is reading correctly, code tips are displayed, imports are automatically added etc.... What you need to do is update Visual Studio Code to the latest version. How did I come across this? Simply by trying to execute the 'Java: Clean Java Language Server Workspace' command I got the message "command 'java.clean.workspace' not found" after a moment's thought, I deduced that the 'Language Support for Java(TM) by Red Hat' plug-in must not be working. The only reason for this may be an outdated version of Visual Studio Code. In my case I upgraded it by selecting the 'code' package to upgrade in the Synaptic package manager on a Debian system and clicking 'Apply'. After updating VS Code and starting it, the 'Language Support for Java(TM) by RedHat' plugin requests that VS Code be restarted again. After restarting, everything works like a charm. The cause of the problem is that I have updated the plugins but not updated Visual Studio Code itself. Lesson learned: if you're updating plugins, check if a VS Code update is available, as the new plugin may require it.
![image](https://github.com/redhat-developer/vscode-java/assets/39669541/40377739-6279-4dda-a4ef-0199b2a9773d)

**Comment by jyothibai** [comment]
![Screenshot (24)](https://github.com/redhat-developer/vscode-java/assets/158141886/7a671e92-38d1-44c5-bd5b-7d50f2a0cb18)

**Comment by jyothibai** [comment]
same issue comes again

**Comment by jyothibai** [comment]
if any solution plz.
